### DISCUSSION TITLE
react-script-editor-on prem gulp await error

### DISCUSSION THREAD

**Discussion by MaPi02** [pnp/sp-dev-fx-webparts#4495] [Q&A]
Hello,

I'm trying to build the react-script-editor-on premise for SharePoint SE. I'm using nodejs v8.17.0 (tried other versions too) and getting the error below.

Steps I did: 

Ahead of npm install i ran the spfx doctor with nodejs v18.19.0 and test for SP2019 (SE not available?) (switched with nvm use) and got this:

√ SharePoint Framework v1.4.1
√ Supported in SP2019
× Node v18.19.0 found, v^6 || ^8 required
√ yo v3.1.1
√ gulp-cli v2.3.0
√ bundled typescript used

Recommended fixes:

- Install Node.js v8
-
switched back to nodejs v8.17.0, ran npm install

> found 305 vulnerabilities (9 low, 158 moderate, 125 high, 13 critical)

tried to go the deploy path: 

gulp clean returns this: 

sp-dev-fx-webparts\samples\react-script-editor-onprem\node_modules\@microsoft\spfx-heft-plugins\lib\plugins\devDeployPlugin\SpDevDeploy.js:165
            for await (const file of filesIterable) {
                ^^^^^

SyntaxError: Unexpected reserved word
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)

I suspect that it is a version problem? Any ideas what I'm doing wrong? Can you help me how to fix this? Thanks in advance.

**Comment by phoboss93** [comment]
I have the same issue. Why does it happen? Are dependencies too old?
